src/inventory_client/inventory_client.go

```diff
@@ -219,14 +219,20 @@ func (c *inventoryClient) GetCluster(ctx context.Context) (*models.Cluster, erro
 }
 
 func (c *inventoryClient) GetEnabledHostsNamesHosts(ctx context.Context, log logrus.FieldLogger) (map[string]HostData, error) {
-	return c.GetHosts(ctx, log, []string{models.HostStatusDisabled})
+	hosts, errorhosts, err := c.GetHosts(ctx, log, []string{models.HostStatusDisabled})
+	for k, v := range errorhosts {
+		hosts[k] = v
+	}
+	return hosts, err
 }
 
-func (c *inventoryClient) GetHosts(ctx context.Context, log logrus.FieldLogger, skippedStatuses []string) (map[string]HostData, error) {
+func (c *inventoryClient) GetHosts(ctx context.Context, log logrus.FieldLogger, skippedStatuses []string) (map[string]HostData, map[string]HostData, error) {
```

Comment:
i don't like the idea to return error hosts as separate var

Reply:
i think u can create func that gets list of error hosts and return list of them. Or lets create function that do what u want specifically. I think it is strange behavior for GetHosts

Reply:
or map in our case

Reply:
DONE